### ISSUE TITLE
Could you Dockerise this project such that it can run regardless of environment? Thanks

### ISSUE THREAD

**Issue by jiacheng-0** [duo-labs/cloudmapper#890]
Please mention the following:
- What command was run?
- Are you working out of a virtualenv environment, Docker, or something else?

sudo apt-get install autoconf automake libtool python3.7-dev python3-tk jq awscli
on Ubuntu 20.04 LTS (WSL)

No virtual env, just installation

**Comment by w0rmr1d3r** [comment]
Hello @jiacheng-0 ,

It actually exists this -> https://github.com/duo-labs/cloudmapper#using-a-docker-container

Have you checked it out?

Cheers!

**Comment by chrisjohnson00** [comment]
> Hello @jiacheng-0 ,
> 
> It actually exists this -> https://github.com/duo-labs/cloudmapper#using-a-docker-container
> 
> Have you checked it out?
> 
> Cheers!

The container doesn't build... so far here's the error's I've seen:

 - `python3.7-dev` isn't a package you can install in `3.7-slim` by default, there's likely some apt repos that are needed to be added to enable this.

Then there's this:
```
ERROR: Cannot install -r requirements.txt (line 5) and urllib3==1.26.5 because these package versions have conflicting dependencies.

The conflict is caused by:
    The user requested urllib3==1.26.5
    botocore 1.17.6 depends on urllib3<1.26 and >=1.20; python_version != "3.4"

```

So it looks like someone hasn't tested this enough.

**Comment by w0rmr1d3r** [comment]
If it's something regarding conflicts or dependencies in the requirements.txt file, there are a lot of open issues about that.
Check them as well, since this error:
```
The conflict is caused by:
    The user requested urllib3==1.26.5
    botocore 1.17.6 depends on urllib3<1.26 and >=1.20; python_version != "3.4"
```

Has already been fixed in other PRs, yet not merged.

**Comment by chrisjohnson00** [comment]
> Has already been fixed in other PRs, yet not merged.

Yes, that's 1 of 2 things I had to fix to get the container to build.
